MobileWeather/ImageProcessor.m

```diff
@@ -31,9 +31,25 @@ - (instancetype)init {
     return self;
 }
 
+- (UIImage *)imageByName:(NSString *) imageName {
```

Comment:
```suggestion
- (UIImage *)imageByName:(NSString *)imageName {
```

Based on moving the app to iOS 13.0+, I don't think this method will be necessary anymore?